src/components/cuisine/Cuisine.js

```diff
@@ -1,39 +1,78 @@
 import React, { Component } from 'react'
 import BackButton from '../common/BackButton'
 import NextButton from '../common/NextButton'
+import SelectionButton from '../common/SelectionButton'
 
 
 export class Cuisine extends Component {
-    continue = e => {
-        e.preventDefault();
-        this.props.nextStep();
-        }
-
-   
+    constructor(props) {
+        super(props);
+        this.state = {
+            chinese: 0,
+            korean: 0,
+            western: 0,
+            japanese: 0
+        };
+        this.handleInputChange = this.handleInputChange.bind(this);
+    }
+    handleInputChange(event) {
+        const target = event.target;
+        const value = target.type === 'checkbox' ? target.checked : target.value;
+        const name = target.name;
+    
+        this.setState({
+          [name]: value
+        });
+    }  
+     
 
     render() {
-        const { handleChange } = this.props;
         return (
             <div>
                 <p> Me no like this</p>
-                <form >
-                        <select id="cuisineVeto" name="cuisineVeto" multiple onChange = {this.props.handleChange('cuisineVeto')}>
-                            <option value="1">Chinese</option>
-                            <option value="2">Western</option>
-                            <option value="3">Japanese</option>
-                            <option value="4">Korean</option>
-                        </select>
-                        <br />
-                        <br />
-                        <NextButton 
-                        nextStep = {this.props.nextStep}
-                    />                
-                </form>
+                
+                <br />
+                <label>
```

Comment:
i think u should be creating a component for this (maybe a common CheckboxButton Component?) then u can just create them using a loop.
Then u can manage them using a dictionary

Reply:
can look at how @badgalchenchen implements her RestaurantCard